storage/store.go

```diff
@@ -39,24 +39,19 @@ var _ shared.ArchiveStoragePlugin = (*Store)(nil)
 var _ io.Closer = (*Store)(nil)
 
 func NewStore(logger hclog.Logger, cfg Configuration, embeddedSQLScripts embed.FS) (*Store, error) {
-	db, err := defaultConnector(cfg.Address)
+	cfg.setDefaults()
+	var db *sql.DB
+	var err error
+	db, err = connector(cfg)
```

Comment:
why is `db` and `err` defined above? They can be defined directly here as `db, err := connector(cfg)`